packages/build/src/plugins/list.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+// TODO: replace with `Object.fromEntries()` after dropping Node <12.0.0
+const fromEntries = require('@ungap/from-entries')
+const got = require('got')
+
+const { addErrorInfo } = require('../error/info')
+
+// Retrieve the list of plugins officially vetted by us and displayed in our
+// plugins directory UI.
+// We fetch this list during each build (no caching) because we want new
+// versions of plugins to be available instantly to all users. The time to
+// make this request is somewhat ok (in the 100ms range).
+const getPluginsList = async function ({ testOpts, featureFlags }) {
+  // We try not to mock in integration tests. However, sending a request for
+  // each test would be too slow and make tests unreliable.
+  if (!featureFlags.pluginsList || testOpts.mockPluginsList) {
+    return []
+  }
+
+  const pluginsList = await fetchPluginsList()
+  return normalizePluginsList(pluginsList)
+}
+
+const fetchPluginsList = async function () {
+  try {
+    const { body } = await got(PLUGINS_LIST_URL, { json: true })
+    return body
+  } catch (error) {
+    const errorA = new Error(`Could not fetch the plugins list: ${error.message}`)
+    addErrorInfo(errorA, { type: 'api', location: { endpoint: PLUGINS_LIST_URL } })
+    throw errorA
```

Comment:
If the HTTP request to https://netlify-plugins.netlify.app/plugins.json fails, we fail the build.

An alternative could be to print a warning and return an empty array. However, that would mean either:
  - Not installing the plugins that were about to be installed. That's probably not something we should do. If a plugin was intended to be used and we cannot use it, the build should fail.
  - Installing the latest version of those plugins. This would defeats the strict versioning and vetoing of plugins we do in `netlify/plugins`, so maybe not a good idea either.

Reply:
I think falling back to `GitHub` is a good idea

Reply:
Would it also make sense to publish this package with a local copy of `plugins.json`? 
We fetch the latest version during publish. 
While not always being up to date that will always work as a fallback. 
Downside is that it can lead to unexpected results when a rebuild might use older plugins. Which the current approach a rebuild might use newer plugins which should be backwards compatible.

Reply:
Solution with @erezrokah discussed on Slack:
  - We first fetch the Netlify Site
  - It this fails (and this should not), we print a warning message. Then, we `require('@netlify/plugins')` and use this.
  - When `@netlify/plugins` is published, Renovate automate includes it in the Monday dependency upgrade, i.e. the only additional maintenance this creates is to run `npm publish` on new releases of `netlify/plugins`